packages/engine-ui/src/components/forms/inputs/AddressSearchInput.js

```diff
@@ -47,12 +70,35 @@ const Component = ({
   const handleDropdownSelect = (event, address) => {
     event.persist()
     setShowDropdown(false)
-
-    return onChangeHandler({
+    setShowSaveFavorite(!isFavorite(address.name))
+    const selected = {
       ...address,
       name: `${address.name}, ${address.county}`,
       street: address.name,
-    })
+    }
+    setSelectedAddress(address)
+    return onChangeHandler(selected)
+  }
+
+  const saveAsFavorite = () => {
+    const favoriteAddresses = getFavoriteAddresses()
+
+    if (isFavorite(selectedAddress.name)) return false
+
+    const displayName = prompt(
+      'Namn på denna favoritposition',
+      `${selectedAddress.name}, ${selectedAddress.county}`
+    )
+    if (!displayName) return false
+    selectedAddress.displayName = displayName
+
+    localStorage.setItem(
+      'favoriteAddresses',
+      JSON.stringify(favoriteAddresses.concat([selectedAddress]))
+    )
+
+    setShowSaveFavorite(false)
+    return false
```

Comment:
Do we need to return false or anything?

Reply:
@momentiris I actually don't know. (bad habit probably from not allowing default browser behavior)
I try without it.